tdrs-frontend/src/components/STTComboBox/STTComboBox.jsx

```diff
@@ -12,36 +13,83 @@ import ComboBox from '../ComboBox'
  * @param {function} handleBlur - Runs on blur of combo box element.
  * @param {function} error - Reference to stt errors object.
  */
+
 function STTComboBox({ selectStt, selectedStt, handleBlur, error }) {
-  const sttList = useSelector((state) => state?.stts?.sttList)
+  const sttReduction = useSelector((state) => state?.stts)
```

Comment:
minor point but i'd go with something like `sttListRequest` for the variable name to indicate what's present in the data structure rather than where in the redux stack it comes from